### ISSUE TITLE
Main Page that's shown in the ReadMe doesn't exist.

### ISSUE THREAD

**Issue by BrandonDRoy** [larslockefeer/obsidian-plugin-todo#83]
There is no helpful indications in the ReadMe docs that provide information about how to get the actual page. Unfortunately this is barely usable due to that.

**Comment by lorenzobrasil** [comment]
Yes, i really don't know how to use this plugin. The side tab shown on the plugin page on Obsidian just don't exist when you download and enable the plugin.
I don't know if this is just broken, or buggy, or I just haven't discovered yet how to use that. Should exist some documentation on how it's supposed to be used.

**Comment by MaWaNo** [comment]
Same for me. Look here:
[https://github.com/larslockefeer/obsidian-plugin-todo/issues/69]

**Comment by julianehmer** [comment]
[https://github.com/https://github.com/larslockefeer/obsidian-plugin-todo/issues/69] doesn't fix this for me. I see neither an aggregated list of todos nor the task ribbon. I am using TODO Version 0.2.8 with Obsidian 1.12.7 on a Mac
